src/main/kotlin/blackjack/domain/card/Cards.kt

```diff
@@ -0,0 +1,72 @@
+package blackjack.domain.card
+
+import blackjack.domain.player.PlayerStatus
+
+class Cards(_value: List<Card> = emptyList()) {
```

Comment:
```suggestion
class Cards(value: List<Card> = emptyList()) {
```

value로만 쓰셔도 될 것 같습니다. 저는 class 인자에 생성자에 대해서는 `_`를 사용하면 가독성을 떨어트리게 하는 부분이 있더라구요.